packages/techdocs-common/src/stages/publish/helpers.ts

```diff
@@ -23,12 +23,15 @@ import recursiveReadDir from 'recursive-readdir';
  * Helper to get the expected content-type for a given file extension. Also
  * takes XSS mitigation into account.
  */
-const getContentTypeForExtension = (ext: string): string => {
+export const getContentTypeForExtension = (
+  ext: string,
+  overrideMimeType: boolean = true,
```

Comment:
this allows us to still upload the files using their actual MIME type, but when serving through Techdocs we convert the unsafe ones to `text/plain` and prevent middle man attacks